src/py-opentimelineio/opentimelineio/adapters/svg.py

```diff
@@ -138,207 +205,347 @@ def contract(self, distance):
         self.height -= 2.0 * distance
 
 
-LCARS_CHAR_SIZE_ARRAY = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-                         0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-                         0, 0, 17, 26, 46, 63, 42, 105, 45, 20, 25, 25, 47, 39, 21,
-                         34, 26, 36, 36, 28, 36, 36, 36,
-                         36, 36, 36, 36, 36, 27, 27, 36, 35, 36, 35, 65, 42, 43, 42,
-                         44, 35, 34, 43, 46, 25, 39, 40,
-                         31, 59, 47, 43, 41, 43, 44, 39, 28, 44, 43, 65, 37, 39, 34,
-                         37, 42, 37, 50, 37, 32, 43, 43,
-                         39, 43, 40, 30, 42, 45, 23, 25, 39, 23, 67, 45, 41, 43, 42,
-                         30, 40, 28, 45, 33, 52, 33, 36,
-                         31, 39, 26, 39, 55]
-image_width = 1000.0
-image_height = 660.0
-font_family = "Roboto"
-lines = ['']
-
-
-def convert_point_to_svg_coordinates(point=Point(0, 0)):
-    y = image_height - point.y
+def convert_point_to_svg_coordinates(point=Point(0, 0), config=None):
```

Comment:
This might be silly, but I had to use None because config was the last argument and since the first argument has a default value I had to give a default value to config.

Reply:
In that case, since this isn't an API in usage, I'd put `config` as a positional argument in from of `point` in that case. Either way, I think this note is superseded by another about avoiding passing the entire config object in.